src/components/Input/Input.jsx

```diff
@@ -0,0 +1,141 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { v4 as uuid } from 'uuid';
+import './Input.scss';
+
+const Input = (props) => {
+  const {
+    className,
+    id,
+    isDisabled,
+    label,
+    onBlur,
+    onChange,
+    onFocus,
+    placeholder,
+    type,
+    value,
+  } = props;
+
+  const handleChange = (event) => {
+    onChange(event);
+  }
+
+  const handleFocus = (event) => {
+    if (onFocus) onFocus(event);
+  }
+
+  const handleBlur = (event) => {
+    if (onBlur) onBlur(event);
+  }
+
+  const classes = classNames(
+    className,
+    'input',
+    // { @TODO Placeholder until size/color props are added.
+    //   [`font-size-${size}`]: size,
+    //   [`font-color-${color}`]: color,
+    // }
+  );
+
+  const inputId = id || uuid();
+
+  const renderInput = () => (
+    <input
+      className={classes}
+      disabled={isDisabled}
+      id={inputId}
+      onBlur={handleBlur}
+      onChange={handleChange}
+      onFocus={handleFocus}
+      placeholder={placeholder}
+      type={type}
+      value={value}
```

Comment:
should add `required aria-required="true"` attributes if `isRequired`

https://dequeuniversity.com/tips/identify-required-elements